jumpscripts/maintenance/fix_rebooted_nodes.py

```diff
@@ -0,0 +1,80 @@
+from JumpScale import j
+import time
+
+descr = """
+This script ensures all vms and router oses survive a node reboot.
```

Comment:
survive is not really the correct phrasing I think, maye rephrase to get started agian?